test/controllers/PublishedControllerSpec.scala

```diff
@@ -69,6 +74,7 @@ class PublishedControllerSpec extends UnitSpec with GuiceOneAppPerSuite {
 
       "return expected content" in new ValidGetTest {
         private val result = target.get(validId)(getRequest)
+        println(contentAsJson(result))
```

Comment:
I think we can remove the println statement now.

Reply:
will od